includes/class-wc-payments.php

```diff
@@ -1712,4 +1714,52 @@ public static function enqueue_assets_script() {
 			]
 		);
 	}
+
+	/**
+	 * Shows an alert notice for Hungarian, Sweden, and Czech Republic merchants on WooCommerce 7.4 and below
+	 */
+	public static function wcpay_show_old_woocommerce_for_hungary_sweden_and_czech_republic() {
+		$currencies        = [ 'HUF', 'SEK', 'CZK' ];
+		$store_currency    = get_woocommerce_currency();
+		$should_show_error = in_array( $store_currency, $currencies, true );
+
+		if ( ! defined( 'WC_VERSION' ) || ! version_compare( WC_VERSION, '7.8', '<' ) || ! $should_show_error ) {
+			return;
+		}
+
+		$notice = '';
+
+		switch ( $store_currency ) {
+			case 'HUF':
+				/* translators: %1$s: The current WordPress version used by the store */
+				$notice = __( 'The WooCommerce version you have installed is not compatible with WooCommerce Payments for a Hungarian business. Please update WooCommerce to version 7.8 or above (you are using %1$s). You can do that via the <a1>the plugins page.</a1>', 'woocommerce-payments' );
+				return;
```

Comment:
I think it should be `break;` instead of `return;`

Reply:
Oh THANK YOU! This is why it wasn't working and I wasn't sure what's happening. I changed it to break and now it's working perfectly! [d68b859](https://github.com/Automattic/woocommerce-payments/pull/6350/commits/d68b859c069f9b81d19ce6c73458481f66781295)